docs/en/latest/plugins/prometheus.md

```diff
@@ -27,439 +27,471 @@ description: This document contains information about the Apache APISIX promethe
 
-```yaml title="conf/config.yaml"
-plugin_attr:
-  prometheus:
-    default_buckets:
-      - 15
-      - 55
-      - 105
-      - 205
-      - 505
-```
+The following are labels for a non-exhaustive list of APISIX metrics and their descriptions.
 
-### Specifying `expire`
+### Labels for `apisix_http_status`
 
-`expire` sets the expiration time of `apisix_http_status`, `apisix_bandwidth`, and `apisix_http_latency` metrics in seconds. When set to 0, metrics will not expire.
+The following labels are used to differentiate `apisix_http_status` metrics.
 
-Here is a configuration example:
+| Name   | Description                                                                                                                   |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------- |
+| code         | HTTP response code returned by the upstream node.                                                                            |
+| route        | ID of the Route that the HTTP status originates from when `prefer_name` is `false` (default), and name of the Route when `prefer_name` to `true`. Default to an empty string if a request does not match any Route.                         |
+| route_id     | Available only in Enterprise. ID of the Route that the HTTP status originates from regardless of the `prefer_name` setting.                        |
+| matched_uri  | URI of the Route that matches the request. Default to an empty string if a request does not match any Route.                              |
+| matched_host | Host of the Route that matches the request. Default to an empty string if a request does not match any Route, or host is not configured on the Route.                             |
+| service      | ID of the Service that the HTTP status originates from when `prefer_name` is `false` (default), and name of the Service when `prefer_name` to `true`. Default to the configured value of host on the Route if the matched Route does not belong to any Service. |
+| service_id   | Available only in Enterprise. ID of the Service that the HTTP status originates from regardless of the `prefer_name` setting. |
+| consumer     | Name of the Consumer associated with a request. Default to an empty string if no Consumer is associated with the request.                       |
+| node         | IP address of the upstream node.                                                                                          |
+| gateway_group_id | Available only in Enterprise. ID of the gateway group that the HTTP status originates from. |
+| instance_id | Available only in Enterprise. ID of the gateway instance that the HTTP status originates from. |
+| api_product_id | Available only in Enterprise. Product ID that the HTTP status originates from. |
```

Comment:
Remove all the labels that are `Available only in Enterprise.` There are 15 of them.